server/lib/configuration.js

```diff
@@ -283,6 +283,32 @@ var conf = module.exports = convict({
       doc: 'Location of "report-uri"',
       default: '/_/csp-violation',
     }
+  },
+  marketing_email: {
+    api_url: {
+      doc: 'User facing URL of the Marketing Email Server',
+      format: 'url',
+      default: 'http://127.0.0.1:1114',
+      env: 'FXA_MARKETING_EMAIL_API_URL'
+    },
+    preferences_url: {
+      doc: 'User facing URL where a user can manage their email preferences',
+      format: 'url',
+      default: '',
+      env: 'FXA_MARKETING_EMAIL_PREFERENCES_URL'
+    }
+  },
+  basket: {
```

Comment:
need development configuration for this.
I suggest we run basket on port 10140, default is 8000 and collides with the auth server locally.